conf/default.toml

```diff
@@ -63,6 +63,8 @@ background_color = "#fdfbff"
 # Note: if the GeoJSON object contains multiple features, only the geometry of the first one will be kept!
 # filter_geometry_url = "https://my.domain.org/assets/boundary.geojson"
 # filter_geometry_data = '{ "coordinates": [...], "type": "Polygon" }'
+# available filters: 'organisation', 'format', 'publicationYear', 'standard', 'inspireKeyword', 'topic', 'spatial', 'license'
+custom_filters = ['organisation', 'format', 'publicationYear', 'standard', 'inspireKeyword', 'topic', 'spatial', 'license']
```

Comment:
```suggestion
# The advanced search filters available to the user can be customized with this setting.
# The following fields can be used for filtering: 'publisher', 'format', 'publicationYear', 'standard', 'inspireKeyword', 'topic', 'isSpatial', 'license'
# advanced_filters = ['publisher', 'format', 'publicationYear', 'topic', 'spatial', 'license']
```

Note: since this is an optional key, it should be commented out in the toml file and the default value should be written in the app (datahub in this case).